README.md

```diff
@@ -370,6 +370,18 @@ const config = {
      * Default: Infinity
      */
     maxCpu: 85,
+
+    /** This applies only if `maxCpu` is defined.
+     * See `maxPause` in the module `too-hot`, for its definition.
+     * Default: 300
```

Comment:
180

Reply:
or is it the default of too-hot?

Reply:
Check line 12 in log.js, I kept it at 300. That's because 300 is good in CI. 180 is good for mobile.

Reply:
Ah right, this is from the README :)